unittest.cfg

```diff
@@ -1,13 +1,19 @@
 [unittest]
 verbose = 2
-plugins = flufl.testing.nose
+plugins =
+    nose2.plugins.prof
```

Comment:
Stop using `flufl.testing.nose` because it conflicts with `nose2.plugins.prof`, and from my _personal_ POV profiling is more useful that style enforcement anyways.